### PR TITLE
Task/create cross sell component

### PR DESCRIPTION
**Pull request by gomezjuliana**
## Description
A component that we can use to draw up a cross-selling page

## Screenshots (if appropriate):
http://hi.buffer.com/4b239df7b4f2

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style and guidelines of this project. 
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.*
- [x] I have performed a self-review of my own code
- [x] I have tested my changes/additions in the latest Chrome, Firefox, and Safari.
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] All new and existing tests passed.*
- [ ] I have performed the accessibility audit of my UI changes according to the accessibility doc. 
- [ ] [Buffer Engineers] Someone from the Design team reviewed and approved my changes
- [ ] [Buffer Engineers] I have notified the BDS team of my changes in the #proj-design-system Slack channel

### CODE REVIEW

**Comment by gomezjuliana** [comment]
just a comment on this one.. super curious how it feels to have the texts in a `texts,js` file 

having it this way also shoot an error in the tests:
jest-auto-snapshots > CrossSell › Matches snapshot when passed all props
TypeError: Cannot read property 'title' of undefined

would love to hear your thoughts on this! @ssvmvss @ivanaszuber @hamstu @federicoweber

**Comment by ivanaszuber** [comment]
Oh, that's a great one @gomezjuliana ! Hmm, I might be confusing things here, but my hunch was that if we're to have the Cross Sell component implemented in the Account app, that we'd want the text and copy to live there too 🤔 . This way, if we have it in the bufferUI, we'd still need to update the bufferUI package versions in each product every time we make a change to the copy 🤔 .

To be clear, I'm still leaning towards the solution to **have all of this live in the AppShell** 😅 . However, since at this time that's not possible, and we'll likely be making lots of changes to the copy - I'm thinking it would be better to decouple this and move the copy and the images to the Account app. 

Curious how that feels?

**Comment by gomezjuliana** [comment]
Great one @ivanaszuber, that makes a lot of sense! I'll try it with that and update the PR :)

File: src/components/CrossSell/CrossSell.jsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+      <CrossSellStyled>
+        {this.renderHeader()}
+        {this.renderBody()}
+        {this.renderFooter()}
+      </CrossSellStyled>
+    );
+  }
+}
+
+CrossSell.propTypes = {
```
Comment (gomezjuliana) [inline src/components/CrossSell/CrossSell.jsx]: I changed this to just pass the texts.. my idea is that if the Account app is handling which product we're cross selling, the component doesn't necessarily need the texts.. curious how this feels @ivanaszuber and @ssvmvss

Comment (ivanaszuber) [reply src/components/CrossSell/CrossSell.jsx]: This feels great to me @gomezjuliana ❤️ ! Not sure if we might be able to come up with a more explicit prop name, than `texts` , but I don't really have any great suggestions either 😅

File: src/components/CrossSell/CrossSell.jsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+  renderBody = () => {
+    const { texts } = this.props;
+
+    return (
+      <CardsContainer>
+        {texts.cards.map(card => (
+          <Card>
+            <img src={card.image} alt="kitty" />
+            <CustomH3>{card.title}</CustomH3>
```
Comment (ivanaszuber) [inline src/components/CrossSell/CrossSell.jsx]: Also, curious what is this magical `CustomH3` component 🤔 ?

Comment (gomezjuliana) [reply src/components/CrossSell/CrossSell.jsx]: hahahahahaha oh noooo! Great catch @ivanaszuber , thanks for pointing it out!

**Pull request merged**